src/Traits/FactoryTrait.php

```diff
@@ -144,19 +144,31 @@ public static function minValue()
      */
     public static function create($year = null, $month = null, $day = null, $hour = null, $minute = null, $second = null, $tz = null)
     {
-        $year = ($year === null) ? date('Y') : $year;
-        $month = ($month === null) ? date('n') : $month;
-        $day = ($day === null) ? date('j') : $day;
+        $now = isset(static::$testNow) ? static::$testNow->getTimestamp() : time();
+        $defaults = array_combine(
+            [
+                'year',
+                'month',
+                'day',
+                'hour',
+                'minute',
+                'second',
+            ],
+            explode('-', date('Y-n-j-G-i-s', $now))
```

Comment:
Does applying the timezone in the end mean to alter the time from the testNow?

The way it works now is to take the time from testNow and assume it is from the passed timezone. That is, you get visually the same time, but it belongs to a different timezone. Thus, it is actually a different time.